portfolio/src/main/webapp/gallery.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Lorraine Lyu's Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+  </head>
+  <body>
+    <div id="content">
+      <ul id="navbar">
+        <li class="nav-button"><a href="index.html" class="nav-link">Home</a></li>
+        <li class="nav-button"><a href="introduction.html" class="nav-link">Introduction</a></li>
+        <li class="nav-button"><a href="projects.html" class="nav-link">Projects</a></li>
+        <li class="nav-button current"><a href="gallery.html" class="nav-link">Gallery</a></li>
+      </ul>
+      <div class="main">
+        <h1>My Image Gallary</h1>
+        <div class="slides">
+          <button class="btn-left" onclick="flip(-1)"><</button>
+          <button class="btn-right" onclick="flip(1)">></button>
```

Comment:
Was the js for this already merged? Just making sure I'm not missing a file.

Reply:
No. I haven't finished the script :)